contacts.html

```diff
@@ -712,9 +679,9 @@ <h2>Contacts</h2>
 						</tr>
 						<tr>
 							<td class='tg-i6ua'>Drama Club<br></td>
```

Comment:
Add three more in Dance club
Samarth Neema b19112
Shivani Pandey b19266
Aarushi Gajri b19001